shared/feature/src/commonMain/kotlin/app/futured/kmptemplate/feature/domain/CounterUseCase.kt

```diff
@@ -9,16 +9,21 @@ import kotlinx.coroutines.isActive
 import org.koin.core.annotation.Factory
 import kotlin.time.Duration
 
+/**
+ * Emits increment in intervals specified by [Args.interval].
+ */
+internal fun interface CounterUseCase : FlowUseCase<CounterUseCase.Args, Long> {
+    data class Args(val interval: Duration)
+}
```

Comment:
Je nejaká výhoda v tvorbe interfacov pre usecases? Príde mi to vcelku zbytočný boilerplate

Reply:
Yes, see https://github.com/futuredapp/kmp-futured-template/pull/125#discussion_r2149390743